spark/src/main/kotlin/com/adevinta/spark/components/chips/SuggestionChip.kt

```diff
@@ -87,6 +79,14 @@ import androidx.compose.material3.SuggestionChip as MaterialSuggestionChip
  * for this chip. You can create and pass in your own `remember`ed instance to observe
  * [Interaction]s and customize the appearance / behavior of this chip in different states.
  */
+@Deprecated(
+    message = "Use one of the options: ChipOutlined, ChipFilled, ChipTinted, ChipDashed",
```

Comment:
Material Suggestion Chip is closer to Outlined than the other styles variant so we should recommend this one only instead